incipyt/project/meta_variables.py

```diff
@@ -60,6 +60,8 @@ class Variable:
     default: Any = ""
     prompt: bool = True
     required: bool = False
+    help: str = ""  # noqa: A003
+    export: bool = True
```

Comment:
Unused yet, the idea was that in case one to generate a basic configuration, the `export=False` keys shouldn't be exported to the configuration, but we only create an empty file for now so I guess that's for a future PR